components/list/list.js

```diff
@@ -64,13 +71,65 @@ class List extends LitElement {
 		return [layout, specialCases];
 	}
 
+	firstUpdated(changedProperties) {
+		super.firstUpdated(changedProperties);
+		this._checked = {};
+		this.addEventListener('d2l-list-item-checked', this._onlistItemChecked.bind(this));
+	}
+
+	itemsChecked() {
+		return Object.keys(this._checked).filter(ref => this._checked[ref]);
```

Comment:
This method name sounds like it does something, but really it's a getter that returns the keys of the checked items. I'd suggest calling it `getSelectedItemKeys()`.

Reply:
- [ ] Change name of this to `getSelectedItemRefs()`

Changed it to `Refs` since key is specific to how it is being stored at the moment.